README.md

```diff
@@ -1,2 +1,39 @@
 # embedded-common
 Embedded Driver common files
+
+## Clone this repository
+If you really need this repository you can check it out with the following command:
+```
+git clone https://github.com/Sensirion/embedded-common.git
+```
+We recommend to download the repository recursively within the [SGP](https://github.com/Sensirion/embedded-sgp) and [SHT](https://github.com/Sensirion/embedded-sht) projects.
+
+## Repistory content
+* *.c files (CRC calculation function)
+* *.h files (type definitions, prototype declarations for i2c communication,...)
+* hw_i2c folder (stub for implementation of the hardware i2c communication, including sample code for some platforms)
+* sw_i2c folder (stub for implementation of the software i2c communication (bit-banging GPIOs), including sample code for some platforms)
+
+## Where to start?
+It is quite simple to get started with our drivers. Well, at least we hope so ;-).
+The only files you might touch will come from the common-repository. There are some architecture specifics we walk you through.
+
+### sensirion_arch_config.h
+You need to specify the integer sizes on your system. We assume you have the ```<stdint.h>``` file available, so we have an include directive for it. If this is not the case on your platform you need to specify the following types yourself:
+
+* s64 = signed 64bit integer
+* u64 = unsigned 64bit integer
+* s32 = signed 32bit integer
+* u32 = unsigned 32bit integer
+* s16 = signed 16bit integer
+* u16 = unsigned 16bit integer
+* s8 = signed 8bit integer
+* u8 = unsigned 8bit integer
+
+Make sure you specify the whole range (64bit sized integers are used on a 32bit system for some sensor data).
+adi
+### hw_i2c_implementation.c
+In this file you implement the hardware i2c communication on your system provided there is one available. You will find four function stubs that you have to implement out. There is an init, a read, a write and a delay function. For further information please check out some of the sample implementations in the hw_i2c/sample-implementations folder in this repository.
```

Comment:
Add backticks around path for visual distinction -> `hw_i2c/sample-implementations` (and in the next subsection for sw_i2c)

Reply:
I'll do that thanks